frontend/app_flowy/packages/appflowy_editor/lib/src/service/shortcut_event/built_in_shortcut_events.dart

```diff
@@ -112,23 +112,23 @@ List<ShortcutEvent> builtInShortcutEvents = [
   ),
   ShortcutEvent(
     key: 'Cursor begin select',
-    command: 'meta+shift+arrow left',
-    windowsCommand: 'ctrl+shift+arrow left',
-    linuxCommand: 'ctrl+shift+arrow left',
+    command: 'meta+shift+arrow left, shift+home',
```

Comment:
Please remove the unnecessary space after `,`.

Reply:
okay sure.